sandbox/grist/functions/text.py

```diff
@@ -339,9 +341,9 @@ def REGEXREPLACE(text, regular_expression, replacement):
   return re.sub(regular_expression, replacement, text)
 
 
-def REPLACE(old_text, start_num, num_chars, new_text):
+def REPLACE(text, position, length, new_text):
```

Comment:
This is another place with changes outside of comments. For this not to break, all mentions of old_text, start_num, num_chars in this function need to be renamed.

Reply:
for this one, can I add this between ``` ``` like I did for lookupRecords?

Name: REPLACE
Usage: REPLACE(text, position, length, new_text)

Reply:
You could, I think it's better to actually change the variables to use the clearer names (as you did), you just need to change it both in the REPLACE line (where they are first introduced) and in the lines below (inside the function) the where they are used.